services/mock-blockchain/src/server.ts

```
@@ -0,0 +1,199 @@
+      try {
+        const { address, points, reason } = req.body;
+        const success = await this.reputationContract.deductPoints(address, points, reason);
+        res.json({ success });
+      } catch (error) {
+        next(error);
+      }
+    });
+
+    this.app.get('/reputation/level/:points', async (req, res, next) => {
+      try {
+        const points = parseInt(req.params.points, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)